docs/reference/go-gno-compatibility.md

```diff
@@ -61,6 +61,7 @@ rune := rune('a')
 | `func (T1...) T2...`                          | full                   | full (needs more tests)                                    |
 | `*T` (pointers)                               | full                   | full\*                                                     |
 | `chan T` (channels)                           | missing (after launch) | missing (after launch)                                     |
+| `imag`                                        | unsupported            | unsupported                                                |
```

Comment:
the `imag` type doesn't actually exist, the types are `complex64` and `complex128` which are stated above to not be supported. We talk about "imag" indirectly with the reference for `builtin`:

>builtin is a "fake" package that exists to document the behaviour of some builtin functions. The "fake" package does not currently exist in Gno, but [all functions up to Go 1.17 exist](https://pkg.go.dev/builtin@go1.17), except for those relating to complex or channel types.

For searchability, you can add `real` and `imag` in that footnote.